go.sum

```diff
@@ -55,6 +55,7 @@ github.com/moby/moby v1.11.1/go.mod h1:fDXVQ6+S340veQPv35CzDahGBmHsiclFwfEygB/TW
 github.com/pkg/errors v0.8.1/go.mod h1:bwawxfHBFNV+L2hUp1rHADufV3IMtnDRdf1r5NINEl0=
 github.com/pkg/errors v0.9.1 h1:FEBLx1zS214owpjy7qsBeixbURkuhQAwrK5UwLGTwt4=
 github.com/pkg/errors v0.9.1/go.mod h1:bwawxfHBFNV+L2hUp1rHADufV3IMtnDRdf1r5NINEl0=
+github.com/pkg/sftp v1.10.1/go.mod h1:lYOWFsE0bwd1+KfKJaKeuokY15vzFx25BLbzYYoAxZI=
```

Comment:
There seems to be a bunch of new packages, also there are older ones like windows-terminal.
Which make no sense, since we don't support windows at all.

Reply:
Did you by chance update your go version?

Reply:
I use `go1.16.5`. I'll check the package relationship.

Reply:
The windows-terminal comes from package `github.com/longhorn/backupstore` -> `github.com/sirupsen/logrus` -> `github.com/konsorten/go-windows-terminal-sequences`.